hackathon/spacecraft/app.config.ts

```diff
@@ -0,0 +1,131 @@
+import { ExpoConfig } from "@expo/config-types";
+
+import { version } from "./package.json";
+
+// Project constants
+const EAS_PROJECT_ID = "012accc3-4ce5-4bae-9f4d-2f842489f07a";
+const PROJECT_SLUG = "spacecraft";
+const OWNER = "weshipit";
+
+// App production config
+const APP_NAME = "Spacecraft";
+const BUNDLE_IDENTIFIER = "weshipit.today.spacecraft";
+const PACKAGE_NAME = "weshipit.today.spacecraft";
+const SCHEME = "spacecraft";
+
+export default ({ config }: { config: ExpoConfig }): ExpoConfig => {
+  const environment = process.env.APP_ENV || "development";
+  console.log("⚙️ Building app for environment:", environment);
+
+  const { adaptiveIcon, bundleIdentifier, icon, name, packageName, scheme } =
+    getDynamicAppConfig(
+      environment as "development" | "preview" | "production",
+    );
+
+  return {
+    ...config,
+    android: {
+      adaptiveIcon: {
+        backgroundColor: "#ffffff",
+        foregroundImage: adaptiveIcon,
+      },
+      package: packageName,
+      playStoreUrl:
+        "https://play.google.com/store/apps/details?id=weshipit.today.spacecraft",
+    },
+    extra: {
+      eas: {
+        projectId: EAS_PROJECT_ID,
+      },
+      storybookEnabled: process.env.STORYBOOK_ENABLED,
+    },
+    icon: icon,
+    ios: {
+      appStoreUrl:
+        "https://apps.apple.com/fr/app/retail-shake-scanner/id1234567890",
+      bundleIdentifier: bundleIdentifier,
+      supportsTablet: true,
+    },
+    name: name,
+    newArchEnabled: true,
+    orientation: "portrait",
+    owner: OWNER,
+    plugins: [
+      [
+        "app-icon-badge",
```

Comment:
you can delete `app-icon-badge` because we are using `getDynamicAppConfig()` now